src/certgen_certificate_config.cc

```diff
@@ -66,7 +66,22 @@ certgen_certificate_config_test (bool verbose)
     printf (" * certgen_certificate_config: ");
 
     //  @selftest
-    //  Simple create/destroy test
+    //  Simple deserialize test
+    std::string configFilePath (SELFTEST_DIR_RO + std::string ("certificate.config"));
```

Comment:
Test need to be performed with at least one file well formed and one file formed correctly (positive vs negative tests)